integrations/standalone/tests/pageobjects/CodeEditor.ts

```diff
@@ -0,0 +1,50 @@
+import { Locator, Page, expect } from '@playwright/test';
+import { FocusCodeEditorUtil } from '../utils/code-editor-util';
+
+export class CodeEditor {
+  private readonly label: string | undefined;
+  private readonly parentLocator: Locator;
+  private readonly page: Page;
+  private readonly contentAssist: Locator;
+  private readonly value: Locator;
+
+  constructor(page: Page, parentLocator: Locator, activateOnFocus: boolean, label: string | undefined) {
+    this.page = page;
+    this.parentLocator = parentLocator;
+    this.label = label;
+    this.contentAssist = parentLocator.locator('div.suggest-widget');
+    this.value = activateOnFocus ? this.locator() : this.locator().getByRole('textbox');
+  }
+
+  async triggerContentAssist() {
+    await FocusCodeEditorUtil.triggerContentAssist(this.page, this.locator());

```

Comment:
I think after every part is using this pageobject we can move the code of this util classes inside here 👍